src/modules/AnalysisExecution/components/ViewEditInsight/Actions/container.js

```diff
@@ -48,7 +49,7 @@ function mergeProps(stateProps, dispatchProps, ownProps) {
     ...ownProps,
     remove: () => {
       Utils.confirmDelete({
-        message: 'Are you sure want to delete insight?',
+        message: `Are you sure want to delete insight '${stateProps.name}'?`,
```

Comment:
https://english.stackexchange.com/questions/222316/are-you-sure-to-delete-or-are-you-sure-you-want-to-delete

Seems like `Are you sure you want to delete ` is more correct